interactors/spinner.js

```diff
@@ -0,0 +1,4 @@
+import HTML from './baseHTML';
+
+export default HTML.extend('spinner')
+  .selector('div[class^=spinner-]');
```

Comment:
Maybe locator and filters should be added?

Reply:
it's not needed in current code. Code can be extended if it will needed